optimum/intel/neural_compressor/trainer.py

```diff
@@ -681,6 +681,7 @@ def _inner_training_loop(
     def save_model(
         self,
         output_dir: Optional[str] = None,
+        _internal_call: bool = False,

```

Comment:
needed for https://github.com/huggingface/transformers/blob/v4.41.0/src/transformers/trainer.py#L2811